Bio/PDB/PDBIO.py

```diff
@@ -51,11 +51,12 @@ class PDBIO(object):
     Examples
     --------
     >>> from Bio.PDB import PDBParser
-    >>> p=PDBParser()
-    >>> s=p.get_structure("1fat", "1fat.pdb")
+    >>> from Bio.PDB.PDBIO import PDBIO
+    >>> parser = PDBParser()
+    >>> structure = parser.get_structure("1a8o", "PDB/1A8O.pdb")
     >>> io=PDBIO()
-    >>> io.set_structure(s)
-    >>> io.save("out.pdb")
+    >>> io.set_structure(structure)
+    >>> io.save("../../bio-pdb-pdbio-out.pdb")
```

Comment:
Writing files to random directories on the system (outside the Biopython tree is a bad idea). How about:

```
>>> io.save("out.pdb")
>>> import os
>>> os.remove("out.pdb")  # tidy up
```

This is not ideal, but we have similar cases elsewhere, e.g.
https://github.com/biopython/biopython/blob/biopython-173/Bio/SeqIO/SffIO.py#L228